app/models/preserved_object.rb

```diff
@@ -4,6 +4,8 @@
 # represent a specific stored instance on a specific node, but aggregates
 # those instances.
 class PreservedObject < ApplicationRecord
+  # NOTE: The size of the object is the size on disk, so it may vary from machine to machine.
+  # It is not used for fixity checking.
```

Comment:
I'm glad to see this comment, but there's is nothing here about size, so it seems out of place.  I'm wondering how to improve this situation.  Maybe something like:

"NOTE: the size field stored in PreservedObject is approximate, as it is determined from size on disk (which can vary from machine to machine).  This field value should not be used for fixity checking!"